### PR TITLE
Feature/simplify naming conventions

### PR DESCRIPTION
**Pull request by PythooonUser**
## Summary
- Changes naming conventions as described in #65.
- Documentation updates are designated in #67.

### CODE REVIEW

File: tests/test_basic.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def test_receive_privmsg(self):
 
         def verify_event(event):
             self.callback_invoked = True
-            self.assertEqual(event.type, 'TWITCHCHATMESSAGE', "Type should be 'TWITCHCHATMESSAGE'")
+            self.assertEqual(event.type, EventType.MESSAGE, "Type should be " + EventType.MESSAGE)
```
Comment (joshuaskelly) [inline tests/test_basic.py]: This is a good change! 👍

**Review by joshuaskelly — APPROVED** [review]
Looks good to me! 👍

**Comment by joshuaskelly** [comment]
Because we are making API breaking changes I think we should bump the version to 1.0.0 for the next release. 😄 (I've already updated the milestone)

**Comment by PythooonUser** [comment]
I thought about that, too.

**Comment by PythooonUser** [comment]
Uppsa^^

**Pull request merged**